ee/clickhouse/views/test/test_clickhouse_groups.py

```diff
@@ -0,0 +1,35 @@
+from ee.clickhouse.models.group import create_group
+from ee.clickhouse.util import ClickhouseTestMixin
+from posthog.test.base import APIBaseTest
+
+
+class ClickhouseTestGroupsApi(ClickhouseTestMixin, APIBaseTest):
+    def test_property_definitions(self):
+        create_group(
+            team_id=self.team.pk,
+            group_type_index=0,
+            group_key="org:5",
+            properties={"industry": "finance", "name": "Mr. Krabs"},
+        )
+        create_group(team_id=self.team.pk, group_type_index=0, group_key="org:6", properties={"industry": "technology"})
+        create_group(team_id=self.team.pk, group_type_index=1, group_key="company:1", properties={"name": "Plankton"})
+        create_group(team_id=self.team.pk, group_type_index=1, group_key="company:2", properties={})
+
+        response = self.client.get(f"/api/projects/{self.team.id}/groups/property_definitions").json()
+        self.assertEqual(response["0"][0]["name"], "industry")
```

Comment:
You're missing things from your assertion - group type 0 has a `name` property as well.

Use `assertEqual` here on the overall response rather than checking individual components - it makes the response structure more obvious and it's simple enough to do here.